vimeo-networking/src/main/java/com/vimeo/networking/Search.java

```diff
@@ -205,16 +202,27 @@ public static FilterCategory fromString(String text) {
     static final String FILTER_DURATION = "filter_duration";
     static final String FILTER_TYPE = "filter_type";
     static final String FILTER_FEATURED_COUNT = "featured_clip_count";
+    static final String PARAMETER_GET_FACETS = "facets";
 
     public static Call<SearchResponse> search(@NotNull String query, @NotNull FilterType type,
                                               @NotNull ModelCallback<SearchResponse> callback,
                                               @Nullable Map<String, String> refinementMap,
-                                              @Nullable String containerFilter,
+                                              @Nullable List<Facet> facets, @Nullable String containerFilter,
                                               @Nullable String fieldFilter) {
         if (refinementMap == null) {
             refinementMap = new HashMap<>();
         }
         refinementMap.put(FILTER_TYPE, type.getText());
+        if (facets != null) {
+            String facetsToUse = "";
+            for (int i = 0; i < facets.size(); i++) {
+                facetsToUse += facets.get(i).getText();
+                if (i + 1 != facets.size()) {
+                    facetsToUse += ",";
+                }
+            }
```

Comment:
Here's a pretty clever way I just found to comma delimit a list (since it never feels like a very pretty thing): 
```java
String delim = "";
for (Item i : list) {
    sb.append(delim).append(i);
    delim = ",";
}
```

http://stackoverflow.com/questions/668952/clearest-way-to-comma-delimit-a-list-java